src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version() -> VersionNumber
+
+### Purpose ###
+
+Return the version of the underlying CFITSIO library
+
+### Example ###
+
+```julia
+julia> using FITSIO
```

Comment:
I'd prefer to omit the line, because I feel it only adds visual noise in reference documentation like this where the user is already aware that they have to import the library. For narrative documentation pages (once we have them) I think it makes sense to add the line for completeness. But... I don't feel very strongly about it.

Yes, I think we shouldn't bother with jldoctest immediately -- it will be tricky for many functions in FITSIO with opening and closing temporary files. We can always add it later.